packages/styled/components/hoc/withThemes.js

```diff
@@ -21,10 +37,18 @@ const withThemes = (themeMap) => (WrappedComponent) => {
      * Prop indicating which theme to use.
      */
     themeName: PropTypes.oneOf(Object.keys(themeMap)),
+    /**
+     * CSS style.
+     */
+    style: PropTypes.oneOf([
+      PropTypes.array,
```

Comment:
Will this still work if style is an array?

Reply:
It's an issue on the PHP side,

An empty (`[]`) or indexed (`[0=>'', 1=>'']`) array will be an array in JSON, and associate arrays (`['hello'=>'world', 'foo'=>'bar']`) will be an object in JSON. So if `style` is empty or unset on the WP side, it comes over as an array, which throws a propType error.

We're fixing on the WP side, but it's part of a much bigger thing and isn't ready yet. So you're right to call it out, but it's a temporary thing.